### ISSUE TITLE
Don't disconnect printer in LAN mode

### ISSUE THREAD

**Issue by UBaer21** [bambulab/BambuStudio#5312]
**Is your feature request related to a problem? Please describe.**
When switching tabs the "Device" tab often shows "no printer" as it disconnects.
Also don't stop the camera (if liveview when printing is on) when switching tabs.
I often need to check something and when switching back the video is off and sometimes also the printer disconnected 
(the disconnect doesn't happen in the previous version 1.9  before 1.10.1.50)

**Describe the solution you'd like**
choosing my printer again and again and again from the dropdown

**Describe alternatives you've considered**
stop using BambuStudio

**Comment by DarkSwoop** [comment]
Same thing here. Really annoying.

**Comment by UBaer21** [comment]
@DarkSwoop yeah but bug reports are so annoying to fill out.

**Comment by DarkSwoop** [comment]
https://github.com/bambulab/BambuStudio/issues/5315 may be related to this.

**Comment by miikememe** [comment]
Still a bug, macOS bambu v 1.10.0.89

**Comment by nateabele** [comment]
Yeah, this is definitely a bug. I have a P1S and this affects all releases of BambuStudio for the past year+.

It disconnects. **CON**. **STANT**. **LY**.

It disconnects if I switch tabs. It disconnects if I minimize the window. It even disconnects if the window sits idle or is in the background for too long.

Given all the other signals (in software releases, on the forums, and elsewhere) indicating how antagonistic Bambu is towards LAN mode and their incentives towards driving people to their cloud service, I can only assume this is intentional.

If anyone's able to make a patch, and I'm happy to fork it and build / release a working version.

**Comment by mkstephenson** [comment]
Yeah, this has been bugging me for a long time. I would be happy with an option to auto-reconnect the previously selected printer when switching to the device tab. I imagine the connection is auto dropped to save on resources (as annoying as that is).

**Comment by travisterrell** [comment]
I really want this functionality—even if the video turns off. I think the current behavior isn't a bug—it's likely intentional behavior to account for the limited number of connection slots available, just to be extra careful to avoid issues. However, the connection limit is at least 3, so it's probably not that big of a deal unless users have several instances open where they've clicked the Device tab and then minimized it.
